speechbrain/lobes/models/ECAPA_TDNN.py

```diff
@@ -64,6 +66,7 @@ def __init__(
         dilation,
         activation=nn.ReLU,
         groups=1,
+        dropout=0.1,
```

Comment:
I think this is going to make all the pretraiened ECAPA-TDNN models backward incompatible? Normally, at test time, we downscale the activation based on the dropout factor. I think we should initialize it to 0.0

Reply:
Sure, a default of 0 makes sense, will do